.prettierignore

```diff
@@ -80,11 +80,13 @@ stats.html
 shared/build
 **/react-shared
 
-core
 **/public/config/config.js
 backend/backend-production.*
 
 **/temp/**
 
 # monaco editor
 **/vs/loader.js 
+
+**/src/assets/**
```

Comment:
Backend folder has `assets` in `core/assets`, so it would better be:
```
**/assets/**
```
I would additionally add 
```
**/*.chunk.*
```